stdlib/3/xmlrpc/client.pyi

```diff
@@ -0,0 +1,279 @@
+    _use_builtin_types: bool
+
+    def __init__(self, use_datetime: bool = ..., use_builtin_types: bool = ...) -> None: ...
+    def close(self) -> Tuple[_Marshallable, ...]: ...
+    def getmethodname(self) -> Optional[str]: ...
+    def xml(self, encoding: str, standalone: Any) -> None: ...  # Standalone is ignored
+    def start(self, tag: str, attrs: Dict[str, str]) -> None: ...
+    def data(self, text: str) -> None: ...
+    def end(self, tag: str) -> None: ...
+    def end_dispatch(self, tag: str, data: str) -> None: ...
+    def end_nil(self, data: str) -> None: ...
+    def end_boolean(self, data: str) -> None: ...
+    def end_int(self, data: str) -> None: ...
+    def end_double(self, data: str) -> None: ...
+    if sys.version_info >= (3, 6):
+        def end_bigdecimal(self, data: str) -> None: ...
+    def end_string(self, data: str) -> None: ...
+    def end_array(self, data: str) -> None: ...
+    def end_struct(self, data: str) -> None: ...
+    def end_base64(self, data: str) -> None: ...
+    def end_dateTime(self, data: str) -> None: ...
+    def end_value(self, data: str) -> None: ...
+    def end_params(self, data: str) -> None: ...
+    def end_fault(self, data: str) -> None: ...
+    def end_methodName(self, data: str) -> None: ...
+
+class _MultiCallMethod:  # undocumented
+
+    __call_list: List[Tuple[str, Tuple[_Marshallable, ...]]]
+    __name: str
+
+    def __init__(self, call_list: List[Tuple[str, _Marshallable]], name: str) -> None: ...
+    def __getattr__(self, name: str) -> _MultiCallMethod: ...
+    def __call__(self, *args: _Marshallable) -> None: ...
+
+class MultiCallIterator:  # undocumented
+
+    results: List[List[_Marshallable]]
+
+    def __init__(self, results: List[List[_Marshallable]]) -> None: ...
+    def __getitem__(self, i: int) -> _Marshallable: ...
+
+class MultiCall:
+
+    __server: ServerProxy
+    __call_list: List[Tuple[str, Tuple[_Marshallable, ...]]]
+
+    def __init__(self, server: ServerProxy) -> None: ...
+    def __getattr__(self, item: str) -> _MultiCallMethod: ...
+    def __call__(self) -> MultiCallIterator: ...
+
+# A little white lie
+FastMarshaller: Optional[Marshaller]
+FastParser: Optional[ExpatParser]
+FastUnmarshaller: Optional[Unmarshaller]
+
+def getparser(use_datetime: bool = ..., use_builtin_types: bool = ...) -> Tuple[ExpatParser, Unmarshaller]: ...
+def dumps(params: Union[Fault, Tuple[_Marshallable, ...]], methodname: Optional[str] = ..., methodresponse: Optional[bool] = ..., encoding: Optional[str] = ..., allow_none: bool = ...) -> str: ...
+def loads(data: str, use_datetime: bool = ..., use_builtin_types: bool = ...) -> Tuple[Tuple[_Marshallable, ...], Optional[str]]: ...
+
+def gzip_encode(data: bytes) -> bytes: ...  # undocumented
+def gzip_decode(data: bytes, max_decode: int = ...) -> bytes: ...  # undocumented
+
+class GzipDecodedResponse(gzip.GzipFile):  # undocumented
+
+    io: io.BytesIO
+
+    def __init__(self, response: IO[bytes]) -> None: ...
```

Comment:
This one just needs `read()` and `close()`, so we can use another ad-hoc protocol.

Reply:
Does it even need close? I don't see close being called on it by the __init__, it's read into the BytesIO and that is what is closed